### ISSUE TITLE
Pressure chamber contents checking

### ISSUE THREAD

**Issue by squirrelkiller** [TeamPneumatic/pnc-repressurized#382]
There is currently no way to test if there is anything inside of a pressure chamber (calling it PC here on out to save my terrible typing) automatically, let alone what is inside of it. Not even basic comparator logic can do so. I would appreciate a way to test the following:

1. If PC has anything in it at all.
2. If the PC has a valid recipe but not enough air.

These could be done with a comparator and differing redstone levels, fancy redstone could then just turn the interface on once given the go ahead until the interior is empty. Maybe even add a redstone interface block for such things.

I would also like to see its contents exposed through the OC and CC compat so they can control the interfaces either through redstone or even changing interface modes.

**Comment by desht** [comment]
Yeah, such comparator support is a good idea.  Probably have a level of 1 if there are any items in the chamber, and 2 if there's a valid recipe but insufficient air.

Note that there's now a redstone module for pressure tubes which allows the tube to transmit redstone (and with an Advanced PCB, carry out various more complex logic operations).
